packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,27 +119,29 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
                 minIncludeTextWidth: 800,
               ),
             ),
+            summaryTreeController?.isExperimentalStoryOfLayoutEnabled == true
+              ? OutlineButton(
+              onPressed: summaryTreeController?.toggleDebugLayoutSummaryEnabled,
+              child: Label(
+                FlutterIcons.lightbulb,
+                'Layout',
+                minIncludeTextWidth: 900,
+              ))
+              : const SizedBox(),
             const Spacer(),
             Row(children: getServiceExtensionWidgets()),
           ],
         ),
         Expanded(
           child: Row(
             children: [
+              summaryTree,
               Expanded(
-                  child: InspectorTree(
-                controller: summaryTreeController,
-              )),
-              Expanded(
-                child: Stack(
-                  children: [
-                    InspectorTree(
-                      controller: detailsTreeController,
-                    ),
-                    if (_expandCollapseSupported) _expandCollapseButtons()
-                  ],
-                ),
-              )
+                child: detailsTreeController?.isExperimentalStoryOfLayoutEnabled == true ?
```

Comment:
since this is repeated, move it as a bool at the top of this method and replace both uses.
e.g.
var enableStoryOfLayout = inspectorTreeController?.isExperimentalStoryOfLayoutEnabled ?? false;